prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -211,7 +211,7 @@ int efa_rdm_ep_post_user_recv_buf(struct efa_rdm_ep *ep, struct efa_rdm_ope *rxe
 	struct efa_mr *mr;
 	int err;
 
-	assert(rxe->iov_count == 1);
+	assert(rxe->iov_count > 0 && rxe->iov_count <= ep->rx_iov_limit);
```

Comment:
I think the current logic in this function relies on iov_count == 1. If we want to touch this, we need to fix the other logic in this function as well

Reply:
better add a TODO here and fix it in a follow-up PR that fix the whole function

Reply:
So remove the `assert()` altogether?

Reply:
I suggest adding a TODO and leave this `assert()` in this function unchanged for now